zksend/zk_bag/sources/zk_bag.move

```diff
@@ -34,6 +35,45 @@ const EClaimAddressNotExists: u64 = 6;
 /// Claims an item that does not exist.
 const EItemNotExists: u64 = 7;
 
+
+/// Event emitted when a new ZkBag is created
+public struct BagCreatedEvent has copy, drop {
+    bag_id: ID,
+    creator: address,
+}
+
+/// Event emitted when an item of type T is added to a ZkBag
+public struct BagItemAddedEvent<phantom T> has copy, drop {
+    bag_id: ID,
+    creator: address,
+}
```

Comment:
I'm not sure if this event is useful without the id of the added item.  In practice items won't be added to links outside the creating transaction, so this is effectively just a duplicate of the create event that let's you count the number of items in the link